src/var/flat.js

```diff
@@ -0,0 +1,18 @@
+define( [
+	"./arr",
+	"./concat"
+], function( arr, concat ) {
+
+"use strict";
+
+// Support: IE 11+, Edge 18+
+// Provide fallback for browsers without Array#flat.
+return arr.flat ? function( array ) {
+
+	// Flat with depth = 1 to give the same result of concat.apply.
+	return arr.flat.call( array, 1 );
```

Comment:
1 is the default, you don't need to specify it. Also, the comment above this line is not really necessary.